src/c++/perf_analyzer/test_request_rate_manager.cc

```diff
@@ -249,18 +276,37 @@ class TestRequestRateManager : public TestLoadManagerBase,
     CheckSequences(params_.num_of_sequences);
   }
 
+  /// Test that the shared memory methods are called correctly
+  ///
+  void TestSharedMemory(uint request_rate, uint duration_ms)
+  {
+    ChangeRequestRate(request_rate);
+    std::this_thread::sleep_for(std::chrono::milliseconds(duration_ms));
+    StopWorkerThreads();
+
+    CheckSharedMemory();
+  }
+
+  std::shared_ptr<ModelParser>& parser_{LoadManager::parser_};
```

Comment:
Is the point of these to make them public?

Reply:
Yes, accessible from the `TEST_CASE` blocks.